Src/Web/Web.Shared.Net/SyntheticUserAgentTelemetryInitializer.cs

```diff
@@ -49,19 +51,26 @@ protected override void OnInitializeTelemetry(HttpContext platformContext, Reque
             {
                 if (platformContext != null)
                 {
-                    var request = platformContext.GetRequest();
-
-                    if (request != null && !string.IsNullOrEmpty(request.UserAgent))
+                    if (platformContext.Items.Contains(SyntheticSourceNameKey))
+                    {
+                        telemetry.Context.Operation.SyntheticSource = platformContext.Items[SyntheticSourceNameKey].ToString();
```

Comment:
Why isn't the right hand side just ```SyntheticSourceName``` ?
(Frankly, the value stored in the ```.Items``` dictionary is irrelevant. It could just be ```null```)